lib/mongo_client.js

```diff
@@ -143,6 +143,7 @@ const { connect, validOptions } = require('./operations/connect');
  * @param {boolean} [options.useNewUrlParser=true] Determines whether or not to use the new url parser. Enables the new, spec-compliant, url parser shipped in the core driver. This url parser fixes a number of problems with the original parser, and aims to outright replace that parser in the near future. Defaults to true, and must be explicitly set to false to use the legacy url parser.
  * @param {DriverInfoOptions} [options.driverInfo] Allows a wrapping driver to amend the client metadata generated by the driver to include information about the wrapping driver
  * @param {AutoEncrypter~AutoEncryptionOptions} [options.autoEncryption] Optionally enable client side auto encryption
+ * @param {boolean} [options.directConnection=false] Enable directConnection
```

Comment:
can we improve this doc string? "Indicates that a client should directly connect to a node without attempting to discover its topology type" or something like that